test/admin-hpa/Rakefile

```diff
@@ -42,8 +44,8 @@ array_of_features.each_with_index do |features, index|
   Cucumber::Rake::Task.new(('feature_set_' + index.to_s).to_sym, "Runs following features #{features.join(' ')}") do |t|
     File.truncate('rerun.txt', 0) if File.exist?('rerun.txt')
     @start_time = Time.now
-    t.cucumber_opts = "--tags 'not @weekend' --format pretty --retry 1 -f rerun -o rerun.txt #{features.join(' ')} DRIVER=#{(ENV['DRIVER']) ? ENV['DRIVER'] : 'headless_chrome'}"
-    t.cucumber_opts << " #{ENV['OPTS']}" unless ENV['OPTS'].nil?
+    # t.cucumber_opts = "--tags 'not @weekend' --format pretty --retry 1 -f rerun -o rerun.txt #{features.join(' ')} DRIVER=#{(ENV['DRIVER']) ? ENV['DRIVER'] : 'headless_chrome'}"
```

Comment:
Why comment this out? There should be a comment, or better still delete the lines entirely if they are no longer required.